docs/docusaurus/sidebars.js

```diff
@@ -80,9 +80,21 @@ module.exports = {
           ]
         },
         {
-          type: 'doc',
-          id: 'core/installation_and_setup/manage_metadata_stores',
-          label: '🚧 Manage Metadata Stores'
+          type: 'category',
+          label: 'Manage Metadata Stores',
+          link: {type: 'doc', id: 'core/installation_and_setup/manage_metadata_stores'},
+          items: [
+            {
+              type: 'link',
+              label: 'Configure Validation Result Stores',
+              href: '/docs/1.0-prerelease/core/installation_and_setup/manage_metadata_stores#configure-validation-result-stores',
+            },
+            {
+              type: 'link',
+              label: 'Configure Expectation Stores',
+              href: '/docs/1.0-prerelease/core/installation_and_setup/manage_metadata_stores#configure-expectation-stores',
+            },
```

Comment:
```suggestion
          {
              type: 'link',
              label: 'Configure Expectation Stores',
              href: '/docs/1.0-prerelease/core/installation_and_setup/manage_metadata_stores#configure-expectation-stores',
            },
            {
              type: 'link',
              label: 'Configure Validation Result Stores',
              href: '/docs/1.0-prerelease/core/installation_and_setup/manage_metadata_stores#configure-validation-result-stores',
            },
```
Flip these to match the order in the right navbar on the page.